docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,235 @@
+``ocean_thermal``. Each test case contains one main step, called
+``ProcessSMB``, ``ProcessBasalMelt`` and ``ProcessThermalForcing``.
+Each step has the local methods (functions) of remapping and
+renaming the original ismip6 data to the format that MALI can incorporate in
+its forward simulations. In remapping the data, all test cases import the
+method ``build_mapping_file`` to create or use scrip files
+of the source (ISMIP6) and destination (MALI) mesh depending on the existence
+of a mapping file. Approximated time for processing a single forcing file
+on Cori (single core) is 2, 7, and 1 minutes for the atmosphere, ocean basal,
+and ocean thermal testcase, respectively.
+
+The ISMIP6 source data can be obtained by directly
+contacting the authors. Reference: https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica#A2.2_Retrieving_datasets_and_uploading_your_model_output
+Once the users get access to the data on Globus (https:www.globus.org), they
+need to create a base directory that is equivalent to the GHub globus endpoints
+(e.g., ``GHub-ISMIP6-Forcing`` for the 2100CE projection protocol and
+``ISMIP6-Projections-Forcing-2300`` for the 2300CE projection protocol.)
+Users also need to manually download the files in the same directory name and
+structure as on the endpoint (including the ``imbie2_basin_numbers_8km.nc``
+file in the ``Ocean_Forcing/imbie2/`` directory, which is provided in the
+globus ``GHub-ISMIP6-Forcing`` endpoint.
+
+In addition to the ISMIP6 forcing files and the basin numbers file, users have
+to download the RACMO modern climatology data `Jan Melchior van Wessem et al
+. (2018) https://tc.copernicus.org/articles/12/1479/2018/#section7>`_.
+if they want to run the ``atmosphere`` test case. The RACMO file is used to
+correct the ismip6 the surface mass balance (SMB) data with the modern
+climatology. Users also have to provide the MALI mesh to which the processed
+forcing files will be saved.
+
+Once the input (source) files are downloaded in the proper directory structure,
+users can configure necessary options (see the below ``config options``
+section for more detail and an example) as well as the ``output_base_path``
+within which the subdirectory called ``atmosphere_forcing/``,
+``basal_melt/`` and ``ocean_thermal_forcing/`` (and further subdirectories that
+match the input file directory structure) are created if the directories do
+not already exist), and processed files will be saved there.
+
+
+config options
+--------------
+
+All three test cases share some set of default config options under the section
+``[ismip6_ais]`` and have separate config options for each test case:
+``[ismip6_ais_atmosphere]``, ``[ismip6_ais_ocean_thermal]``, and
+``[ismip6_ais_ocean_basal]``. In the general config section
+``[ismip6_ais]``, users need to supply base paths to input files and MALI mesh
+file, and MALI mesh name, as well as the model name, climate forcing scenario
```

Comment:
It seems worth mentioning here (as well as in the config files below) that we only support a subset of the models from 2100 and why that choice was made.  (Are the other models only in lower tier experiments?)  Here are some 2100 models I see that we don't seem to support: 2 different `CNRM` models, `MIROC`, `IPSL`, maybe others.

Reply:
Thanks for taking a closer look at the dictionaries, @xylar! The `CNRM` models ,`MIROC` and `IPSL` models are supported in the 2100 projection (https://github.com/MPAS-Dev/compass/blob/7119e28a942b65618e2699b87b00c7dd41110ee8/compass/landice/tests/ismip6_forcing/atmosphere/process_smb.py#L343-L369), and all the experiments listed in the ISMIP6 protocols (e.g., `experiment 1-13` and `exp A1-8` for ISMIP6-2100 and `exp A1-14` (tier1 and tier2) for ISMIP6-2300). I saw that  the `ccsm4_rcp2.6` folder is provided on Ghub ISMIP 2100, but the forcing is not used in any experiments. We can go through it together when we meet to see if I'm missing something.